Package.swift

```diff
@@ -12,11 +12,19 @@
 
 import PackageDescription
 
-let targets: [PackageDescription.Target] = [
+var windowsPlatform: [Platform] = []
+#if os(Windows)
+windowsPlatform.append(.windows)
+#endif
+
+let targets: [Target] = [
   .target(
     name: "SystemPackage",
     dependencies: ["CSystem"],
     path: "Sources/System",
+    cSettings: [
+      .define("_CRT_SECURE_NO_WARNINGS", .when(platforms: windowsPlatform)),
```

Comment:
We can do this unconditionally, it won't impact other targets.  Alternatively, we _could_ clean up the cases to use the Microsoft "extensions" (technically I believe that they are part of Annex G, so they _should_ be portable).

Reply:
Sounds reasonable. Fixed